### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by AndriyRubanskui**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint src/app.js"
```
Comment (anna-kuzii) [inline package.json]: + sass lint

File: public/index.html [hunk 2]
```diff
@@ -2,37 +2,12 @@
...
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
+    <!-- <link rel="stylesheet" href="../src/reset.css"> -->
```
Comment (anna-kuzii) [inline public/index.html]: do we need this comment?

File: src/App.css [hunk 3]
```diff
@@ -1,33 +1,6 @@
...
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+    width: 91%;
```
Comment (anna-kuzii) [inline src/App.css]: 91? why not 100?

File: src/App.css [hunk 4]
```diff
@@ -1,33 +1,6 @@
...
+    height: auto;
+    margin: auto;
+    padding: 10px 0 20px 0;
```
Comment (anna-kuzii) [inline src/App.css]: sass lint should show u error for that

File: src/components/description/index.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import './style.scss';
+
+
+export default function Description() {
+  return (
+    <section className="description">
+    <div className="image-container">
+      <img src={require("../../images/005(png).png")} alt="" className="description-image" />
```
Comment (anna-kuzii) [inline src/components/description/index.jsx]: alt is empty. Why?

File: src/components/description/index.jsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    </div>
+    <article className="description-article">
+      <h2 className="description-article-title">Todd Pownell Trunk Show</h2>
+      <span className="description-article-date">September 12-20, 2015</span>
+      <p className="description-article-par">
+        Quadrum Gallery is thrilled to announce a Todd Pownell Trunk Show
```
Comment (anna-kuzii) [inline src/components/description/index.jsx]: do we need all of this classes? we discussed it

File: src/components/description/style.scss [hunk 7]
```diff
@@ -0,0 +1,74 @@
+@import url('../../fonts.css');
```
Comment (anna-kuzii) [inline src/components/description/style.scss]: why css?

File: src/components/description/style.scss [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+$description-black: #000000;
+$description-seven-grey: #7a7a7a;
+$description-six-grey: #696a67;
```
Comment (anna-kuzii) [inline src/components/description/style.scss]: pls make separate file for vars

File: src/components/description/style.scss [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+  font-size: 16px;
+  color: $description-black;
+}
+
+.description {
+  margin: 30px;
+  text-align: center;
+}
+
+@media (min-width: 1080px) {
```
Comment (anna-kuzii) [inline src/components/description/style.scss]: var for breakpoints

File: src/components/description/style.scss [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+  .description {
+    display: flex;
```
Comment (anna-kuzii) [inline src/components/description/style.scss]: also we discussed that media queries should  be inside selector not vice  versus

File: src/components/footer/footer-article-list/index.jsx [hunk 11]
```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import './style.scss';
+
+export default function Footer_article_list() {
```
Comment (anna-kuzii) [inline src/components/footer/footer-article-list/index.jsx]: pls use arrow function

File: src/components/footer/footer-article-list/index.jsx [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+  return (
+    <ul className="footer-article-list">
+          <li className="footer-article-item">
+            <h4 className="article-title">CONTACT US</h4>
+            <p className="article-par">
```
Comment (anna-kuzii) [inline src/components/footer/footer-article-list/index.jsx]: wow.. u have so much time to create so many classes. pls remove them

File: src/components/footer/footer-article-list/index.jsx [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+          </li>
+          <li className="footer-article-item">
+            <h4 className="article-title">CUSTOMER SERVICES</h4>
+            <ul className="service-list">
+              <li className="service-list-item">F.A.Q.</li>
+              <li className="service-list-item">Contact us</li>
+              <li className="service-list-item">My Account</li>
+              <li className="service-list-item">Gift Cards / Gift Wrapping</li>
+              <li className="service-list-item">Repairs</li>
+              <li className="service-list-item">Returns</li>
```
Comment (anna-kuzii) [inline src/components/footer/footer-article-list/index.jsx]: ahhhh.. u didn't listen to me

File: src/components/footer/footer-article-list/index.jsx [hunk 14]
```diff
@@ -0,0 +1,104 @@
...
+                    className="link-item-link-image"
+                /></a>
+              </li>
+              <li className="link-item">
+                <a href="xcvb" className="link-item-link"
+                  ><img
+                    src="images/pinterest.svg"
+                    alt=""
+                    className="link-item-link-image"
+                /></a>
```
Comment (anna-kuzii) [inline src/components/footer/footer-article-list/index.jsx]: pls make array and use map

File: src/components/footer/footer-article-list/style.scss [hunk 15]
```diff
@@ -0,0 +1,199 @@
...
+  width: 50%;
+  margin: auto;
+  margin-bottom: 30px;
+}
+
+.footer-article-item {
+  flex-basis: 55%;
+  margin-bottom: 30px;
+
+  &:nth-child(4) {
```
Comment (anna-kuzii) [inline src/components/footer/footer-article-list/style.scss]: type?

File: src/components/header/logo/index.jsx [hunk 16]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './style.scss';
+
+export default function Logo() {
+    return (
+        <div className="logo">
+          <a href="zxcv" className="logo-link"
```
Comment (anna-kuzii) [inline src/components/header/logo/index.jsx]: href="zxcv" -- what is the link?

File: src/components/header/logo/style.scss [hunk 17]
```diff
@@ -0,0 +1,6 @@
+@import url('../../../fonts.css');
```
Comment (anna-kuzii) [inline src/components/header/logo/style.scss]: why do u import it here

File: package.json (hunk 1 above)
Comment (AndriyRubanskui) [reply package.json]: ok

File: public/index.html (hunk 2 above)
Comment (AndriyRubanskui) [reply public/index.html]: No)

File: src/App.css (hunk 3 above)
Comment (AndriyRubanskui) [reply src/App.css]: Because i need some free space)

File: src/App.css (hunk 4 above)
Comment (AndriyRubanskui) [reply src/App.css]: But it doesnt do it)

File: src/components/description/index.jsx (hunk 5 above)
Comment (AndriyRubanskui) [reply src/components/description/index.jsx]: Because lint was angry

File: src/components/footer/footer-article-list/index.jsx (hunk 13 above)
Comment (AndriyRubanskui) [reply src/components/footer/footer-article-list/index.jsx]: ohhh...i forgot about it

File: src/components/footer/footer-article-list/index.jsx (hunk 14 above)
Comment (AndriyRubanskui) [reply src/components/footer/footer-article-list/index.jsx]: Hmm...ok

File: src/components/footer/footer-article-list/style.scss (hunk 15 above)
Comment (AndriyRubanskui) [reply src/components/footer/footer-article-list/style.scss]: type)

File: src/components/header/logo/index.jsx (hunk 16 above)
Comment (AndriyRubanskui) [reply src/components/header/logo/index.jsx]: it is no link)

File: src/components/header/logo/style.scss (hunk 17 above)
Comment (AndriyRubanskui) [reply src/components/header/logo/style.scss]: i was afraid it wouldn't work.

**Pull request merged**